frontend/src/static/js/main.js

```diff
@@ -62,35 +62,38 @@ window.app = new window.Vue({
             window.alert('Either the .zip file is corrupted, or you uploaded a .zip file that is not generated '
                 + 'by RepoSense.');
           })
-          .then(() => this.updateReportView().then(() => this.renderTabHash()));
```

Comment:
The original implementation is actually wrong as calling `renderTabHash()` when `updateReportView` encounters an error in parsing the JSON does not make sense.